### PR TITLE
upload description and photo

### PR DESCRIPTION
**Pull request by Cyangray**

### CODE REVIEW

File: content/about/employees/francesco/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+email: francesco@xal.no
+---
+
+Francesco holds a PhD in experimental nuclear astrophysics from the University of Oslo, where he also continued as a postdoc before joining Expert Analytics. The subject of his PhD thesis was the extraction of nuclear properties from experimental data, and their use in astrophysical settings, such as the origin of heavy elements. This required knowledge not only in the underlying nuclear and astrophysics, but also Python programming, statistics, experiment planning and data analysis. He likes solving complex problems, developing algorithms, and enjoys visualizing and presenting data.
+
+<!--more-->
+
+Before his PhD, Francesco studied neutron stars at NTNU in Trondheim, Norway, and plasma physics at UiT in Tromsø, Norway. These are ßboth complex subjects blending topics such as general relativity, thermodynamics, electrodynamics, fluid dynamics, quantum many-body systems and quantum field theory. During his academic journey he developed skills in Python programming, data analysis, simulations of complex nuclear and astrophysical systems, error propagation and data visualisation, as well as teaching, team collaboration and science communication.
```
Comment (anabudisa) [inline content/about/employees/francesco/index.md:15]:
```suggestion
Before his PhD, Francesco studied neutron stars at NTNU in Trondheim, Norway, and plasma physics at UiT in Tromsø, Norway. These are both complex subjects blending topics such as general relativity, thermodynamics, electrodynamics, fluid dynamics, quantum many-body systems and quantum field theory. During his academic journey, he developed skills in Python programming, data analysis, model simulations and data visualisation, as well as teaching, team collaboration and science communication.
```

File: content/about/employees/francesco/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+title: "Francesco Pogliano"
+date: 2024-10-01
+resources:
+  - name: photo
+    src: francesco.jpg
+linkedin: https://www.linkedin.com/in/francesco-pogliano/
+email: francesco@xal.no
+---
+
+Francesco holds a PhD in experimental nuclear astrophysics from the University of Oslo, where he also continued as a postdoc before joining Expert Analytics. The subject of his PhD thesis was the extraction of nuclear properties from experimental data, and their use in astrophysical settings, such as the origin of heavy elements. This required knowledge not only in the underlying nuclear and astrophysics, but also Python programming, statistics, experiment planning and data analysis. He likes solving complex problems, developing algorithms, and enjoys visualizing and presenting data.
```
Comment (anabudisa) [inline content/about/employees/francesco/index.md:11]:
```suggestion
Francesco holds a PhD in experimental nuclear astrophysics from the University of Oslo, where he also continued as a postdoc before joining Expert Analytics. The subject of his PhD thesis was the extraction of nuclear properties from experimental data, and their use in astrophysical settings, such as the origin of heavy elements. He acquired expertise in nuclear physics and astrophysics, but also Python programming, statistics, experiment planning and data analysis. He likes solving complex problems, developing algorithms, and has keen interest in visualizing and presenting data.
```

**Review by anabudisa — COMMENTED** [review]
Very good! :)

Just tiny suggestions from me

File: content/about/employees/francesco/index.md (hunk 1 above)
Comment (Cyangray) [reply content/about/employees/francesco/index.md]: Corrected in new version

File: content/about/employees/francesco/index.md (hunk 2 above)
Comment (Cyangray) [reply content/about/employees/francesco/index.md]: Corrected in new version

**Review by skavhaug — APPROVED** [review]

**Pull request merged**